### ISSUE TITLE
Error `Parent Device is Lost` when launching ruffle

### ISSUE THREAD

**Issue by Ezuharad** [ruffle-rs/ruffle#13414]
### Describe what you were doing

Launching ruffle from the command line causes the program to crash.

### What does the crash message say?

panicked at 'Error in Surface::configure: Validation Error

Caused by:
    Parent device is lost
', /home/ezuharad/.cargo/git/checkouts/wgpu-c26c9ef766f7dc30/c4b8787/wgpu/src/backend/direct.rs:771:18

### Ruffle Version

0.1.0-nightly (25cf77c4a2b4ad616a03811fbb65008bb5186529 2023-10-03)

### Affected platform

Desktop app

### Operating system

Arch Linux Rolling Release [64-bit]

### Additional information

thread 'main' panicked at 'Error in Surface::configure: Validation Error

Caused by:
    Parent device is lost
', /home/ezuharad/.cargo/git/checkouts/wgpu-c26c9ef766f7dc30/c4b8787/wgpu/src/backend/direct.rs:771:18
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
zsh: IOT instruction (core dumped)  ruffle

**Comment by Ezuharad** [comment]
This seems to be an issue with running on Wayland; running on X the program works fine

**Comment by Ezuharad** [comment]
Possible fix for anyone running into the same problem:
Launch ruffle with the environment variable `VK_ICD_FILENAMES=/usr/share/vulkan/icd.d/intel_icd.x86_64.json`.
You might need to install the `lib32-vulkan-intel` and `vulkan-intel` packages for this to work.

**Comment by lolpro11** [comment]
Have the same  problem

**Comment by Ezuharad** [comment]
Was adding the environment able to fix the problem for you @lolpro11?

**Comment by lolpro11** [comment]
> Did adding the environment fix the problem for you @lolpro11?

Nope. I tried and it did not work because I'm on an AMD + NVIDIA system
